modin/core/execution/dask/common/task_wrapper.py

```diff
@@ -39,11 +41,14 @@ def deploy(cls, func, num_returns, kwargs):
             The result of ``func`` splitted into parts in accordance with ``num_returns``.
         """
         client = default_client()
-        remote_task_future = client.submit(func, **kwargs)
-        return [
-            client.submit(lambda l, i: l[i], remote_task_future, i)
-            for i in range(num_returns)
-        ]
+        remote_task_future = client.submit(func, *args, **kwargs)
+        pure = kwargs.get("pure", None)
+        if num_returns != 1:
+            remote_task_future = [
+                client.submit(lambda l, i: l[i], remote_task_future, i, pure=pure)
+                for i in range(num_returns)
+            ]
+        return remote_task_future
 
     @classmethod
     def materialize(cls, future):
```

Comment:
I would probably prefer to rename this method to `get`.

Reply:
I'm for it, only so far it's called that everywhere (including for Ray). Let's do it later and immediately for every engine.

Reply:
ok